sdk/tables/azure-data-tables/src/table_clients.cpp

```diff
@@ -290,7 +290,7 @@ Azure::Response<Models::Table> TableServiceClient::CreateTable(
   auto url = m_url;
   url.AppendPath("Tables");
 
-  std::string jsonBody = Serializers::Create(tableName);
+  std::string jsonBody = Serializers::Create(Azure::Core::Url::Encode(tableName));
```

Comment:
Is the table name in the JSON body URL encoded as well as being JSON encoded?

That is rather strange, IMHO. But I don't know the service very well, so I don't know its expected semantics.

It might be helpful to include a link to the spec which describes why the TableName attribute is URL encoded.

Reply:
done